cli/ioctl/cmd/account/account.go

```diff
@@ -196,3 +204,101 @@ func newAccountByKey(alias string, privateKey string, walletDir string) (string,
 	}
 	return addr.String(), nil
 }
+
+func newAccountByKeyStore(alias, passwordOfKeyStore, keyStorePath string, walletDir string) (string, error) {
+	keyJSON, err := ioutil.ReadFile(keyStorePath)
+	if err != nil {
+		return "", fmt.Errorf("keystore file \"%s\" read error", keyStorePath)
+	}
+	key, err := keystore.DecryptKey(keyJSON, passwordOfKeyStore)
```

Comment:
As you already decrypted private key, can you just reuse newAccountByKey() for the following step?